src/core/TrainController.cpp

```diff
@@ -339,14 +339,16 @@ namespace godot {
         return max_velocity;
     }
 
-    void TrainController::set_axle_arrangement(String p_value) {
+    void TrainController::set_axle_arrangement(const String &p_value) {
         axle_arrangement = p_value;
     }
 
     String TrainController::get_axle_arrangement() const {
         return axle_arrangement;
     }
 
+    void TrainController::set_state(const Dictionary &p_state) {}
```

Comment:
Why? Is this change related to the code cleanup? The state property is READ-ONLY.

Reply:
Method was declared in .hpp file but not in .cpp

So i just have declared it in .cpp